packages/patternfly-4/react-catalog-view-extension/scripts/copyTS.js

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
These aren't var requires? Can this rule be configured for JS files to not complain?

Reply:
@redallen thanks for the callout - these do meet the definition of incorrect code based on this rule: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-var-requires.md

It doesn't look like this rule can be configured - this is an error you'll see a lot of across upcoming PRs